dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/GlobalTagLibraryCache.java

```diff
@@ -6,9 +6,6 @@
  * http://www.eclipse.org/legal/epl-2.0/
```

Comment:
Should we have a 2024 update here in the copyright header since we've made changes to the SPI calls in this class this year?